### PR TITLE
update time and added read me

### PR DESCRIPTION
**Pull request by steverado9**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# 30 days of javascrip challenge
```
Comment (kelvinator07) [inline README.md:1]:
```suggestion
# 30 days of javascript challenge
```

File: app.js [hunk 2]
```diff
@@ -18,11 +18,12 @@ changeColor();
 function dateAndTime() {
     const dateAndTime = document.querySelector('header p');
     let time = new Date();
+    let month = "October";
```
Comment (kelvinator07) [inline app.js]: This should be dynamic, use javascript to get the current month

Comment (steverado9) [reply app.js]: If i get the month, it will be showing as number

File: app.js [hunk 3]
```diff
@@ -18,11 +18,12 @@ changeColor();
 function dateAndTime() {
     const dateAndTime = document.querySelector('header p');
```
Comment (kelvinator07) [inline app.js]: This should be outside the function

File: app.js (hunk 2 above)
Comment (kelvinator07) [reply app.js]: Then use the number to get the month

Comment (steverado9) [reply app.js]: ok sir

**Pull request merged**